client/src/definitions/languages.ts

```diff
@@ -29,6 +29,9 @@ interface LanguagePlugin {
   /** Returns a language-specific editor that handles the UI in a notebook  */
   editor : Editor<EditorState, any>
 
+  /** Returns a language-specific editor that handles the UI in a notebook  */
+  serviceURI? : string
```

Comment:
I don't think this should be here - the fact that the language uses a service should be completely hidden in the implementation. The system should not need to know about this fact.